src/main/java/dev/corestone/lotrings/abilities/PushBackAbility.java

```diff
@@ -26,7 +26,7 @@ public PushBackAbility(LOTRings plugin, Ring ring, String abilityName) {
             this.powerY = plugin.getAbilityDataManager().getAbilityFloatData(abilityName, "vertical-power").doubleValue();
             this.sound = Sound.valueOf(plugin.getAbilityDataManager().getAbilityStringData(abilityName, "sound").toUpperCase());
             this.cooldownManager = new CooldownManager(plugin, this, plugin.getAbilityDataManager().getAbilityFloatData(abilityName, "cooldown-seconds").doubleValue());
-        }catch (Exception e){
+        } catch (Exception e) {
```

Comment:
why

Reply:
Look It was ABSOLUTELY nessisary.

Reply:
Change it back! You broke the entire plugin!